src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -177,6 +190,27 @@ const NumberOfInstancesDropdownInfo: Info = {
   },
 };
 
+// Block-scoped Variables to store information for cost calculation
+let priceMap: Map<string, Map<string, number>>;
```

Comment:
why declare it here? Declare and use where it is initialized

Reply:
Need this to be available outside the function that initializes it, that's why have declared it outside the scope of the initalize function.